views/site/login.php

```diff
@@ -24,7 +24,7 @@
         ],
     ]); ?>
 
-        <?= $form->field($model, 'username') ?>
+        <?= $form->field($model, 'username')->textInput(['autofocus' => 'autofocus']) ?>
```

Comment:
I usually prefer XML form for attributes (`autofocus="autofocus"`). But if you wish, we could use `autofocus`.